Makefile

```diff
@@ -46,6 +46,10 @@ wait-and-validate:
 functests-on-ci: setup-test-cluster feature-deploy feature-wait functests
 
 origin-tests:
+	@echo "Running origin-tests"
+	TESTS_IN_CONTAINER=true ORIGIN_TESTS_FILTER="$(ORIGIN_TESTS_FILTER)" hack/run-origin-tests.sh
+
+skopeo-origin-tests:
```

Comment:
seems like you are adding a makefile target in this commit

Reply:
i just added TESTS_IN_CONTAINER=true because we changed the default to false in the common.sh

Reply:
Isn't this default different from the cnf-tests one? If so, can we stick with the same default (not in container)?
Switching everything to "IN_CONTAINER" is fine as well

Reply:
skopeo-origin-tests run like cnf-tests (I guess you mean the `functests` target ?)
the origin-tests are just to be handy for devs, which I guess will prefer to run in container

you prefer to reverse it? have `origin-tests` run skopeo and add `origin-tests-podman` to run container?

we cant change `functests` otherwise we'll need to introduce something like `functests-on-ci` to use skopeo (bc CI is running in pod, and it is difficult to run podman in a pod)

Reply:
So, the default for cnf-tests is run as a binary.
I'd like the default to be consistent (because consistency is good :-) ), so either both sticking to IN_CONTAINER=false or switching the default IN_CONTAINER to true. If we do the latter, we need to change how u/s CI is called.

Reply:
see that we have two targets, one with containers and the other with the binary

Reply:
Sorry, I read in a rush.
Works for me